truewiki/web_routes.py

```diff
@@ -19,6 +20,7 @@
     preview,
     sitemap,
 )
+
```

Comment:
Guess this newline was unintentional? :)

Reply:
This was the linter, python-black. It wants empty newlines. Do you have a different config? I have been using `.venv-wsl/bin/python -m black -l 120 truewiki`

Reply:
Weird. Might have been a recent update of black, as every PR is validated against black :D So these newlines kinda surprise me .. will check :D